bonobot/basebot.py

```diff
@@ -57,10 +57,10 @@ def __init__(self):
         with open('inchequeable.txt') as f:
             self.triggers = f.read().splitlines()
 
-    def is_relevant(self, type, **kwargs):
+    def is_relevant(self, type, text='', **kwargs):
         if type == 'message':
-            text = kwargs['text'].lower()
-            return any([line.lower() in text for line in self.triggers])
+	    lowcase_text = text.lower()
```

Comment:
this diff looks weird, are you using tabs instead of spaces?

Reply:
I realised midway I was using tabs instead of spaces and switched, but I missed that one.